code.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
```

Comment:
Kindly use linters to the repo. You can geth them from the link belowe to add html and css linters.
https://github.com/microverseinc/linters-config/blob/master/README.md